ModelFarm/src/main/java/io/deephaven/modelfarm/RDMModelFarm.java

```diff
@@ -23,11 +23,11 @@
  * table}. Each row of this table should contain all of the data necessary to populate an instance of {@code DATATYPE},
  * which will then be passed to the {@link ModelFarmBase#model model}.
  *
- * @param <KEYTYPE> The type of the keys (e.g. {@link io.deephaven.modelfarm.fitterfarm.FitScope}).
+ * @param <KEYTYPE> The type of the keys (e.g. {@code io.deephaven.modelfarm.fitterfarm.FitScope}).
```

Comment:
Remember to handle this the same was as the `ModelFarmOnDemand` decision.

Reply:
See above